src/main/java/duke/system/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke.system;
+
+import duke.system.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ListItem;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents an object that interacts with I/O according the file path provided,
+ * imports list from local file with <code>loadData</code> and export using <code>writeData</code> accordingly
+ */
+public class Storage {
+    private final String filePath;
+
+    public Storage(String path) {
+        this.filePath = path;
+    }
+
+    /**
+     * Reads the list and loop till the end of the file and add corresponding tasks.
+     * @return a list of tasks that can be manipulated later
+     * @throws DukeException.IOErrorException
+     */
+    public List<ListItem> loadData() throws DukeException.IOErrorException {
+        List<ListItem> importedList;
+        importedList = new ArrayList<>();
+        try {
+            File f = new File(this.filePath);
+            Scanner s = new Scanner(f); // create a Scanner using the File as the source
```

Comment:
I think writing one lettered variable is not very descriptive. Perhaps you should try more descriptive names such as file and scanner